drupal8.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "drupal8",
+  "plugin_schema": "2",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/SumGuyV5/iocage-plugin-drupal8.git",
+  "pkgs": [
+    "www/apache24",
+    "lang/php72",
+    "archivers/php72-zlib",
+    "lang/php72-extensions",
+    "ftp/php72-curl",
+    "www/mod_php72",
+    "databases/mysql56-server",
+    "databases/mysql56-client",
+    "www/drupal8",
+    "databases/php72-mysqli",
+    "security/php72-hash",
+    "graphics/php72-gd",
+    "databases/php72-pdo_mysql",
+    "www/mod_php72",
+    "drush-php72"
+  ],
+  "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/release_3",
```

Comment:
Would it not make sense to switch this to latest? To ensure the latest update will be catch? Drupal is know for they Security issues.